streamlit_app/requirements.txt

```diff
@@ -1,4 +1,4 @@
-./../go_attack_utils
+-e ./../go_attack_utils
```

Comment:
Why do we need it to be develop mode here? Is there something wrong with the setup.py (missing packages, or package_data)?

Generally seems best to avoid -e outside of development. I feel OK with it for notebooks as you often *are* interactively developing the notebooks, but for a Dockerized app it feels a bit much.

Reply:
I think I was debugging something, added `-e` everywhere, and forgot to revert. Will remove